Resources/Locales/es_es.json

```diff
@@ -0,0 +1,4 @@
+{
+    "descriptionEnglish": "Español (Spanish)",
```

Comment:
The description here should be totally in english. 

e.g.: `"descriptionEnglish": "Spanish (from Spain)` or just *Spanish* is fine.